### PR TITLE
Inputs

### PR DESCRIPTION
**Pull request by Sameony**
Textbox input
image
Validated input
image
Radio and checkbox
image

### CODE REVIEW

**Review by vivekmehto — APPROVED** [review]
looks good.

File: Styles/main.css [hunk 1]
```diff
@@ -107,8 +107,8 @@ body {
 }
 
 .container {
-    width: calc(100% - 25rem);
-    padding-left: 23rem;
+    width: calc(100% - 17rem);
+    padding-left: 10rem;
     color: var(--darc);
     transition-timing-function: ease-in;
     transition: all 0.5s;
```
Comment (kjais1720) [inline Styles/main.css]: you can use the shorthand for transition

File: Templates/Inputs/inputs.html [hunk 2]
```diff
@@ -5,115 +5,256 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Input Mirinda</title>
+    <title>Mirinda Input</title>
```
Comment (kjais1720) [inline Templates/Inputs/inputs.html:8]: The title should have the current page name first then the brand name separated by a `|`

File: Templates/Inputs/inputs.html [hunk 3]
```diff
@@ -5,115 +5,256 @@
...
+        &lt;div class="contentInput"&gt;
+            &lt;div class="boxInput mg-y-sm"&gt;
+                &lt;label for="name" class="boxLabel"&gt;Name&lt;/label&gt;
+                &lt;input type="text" id="name" class="itemInput fs-l"&gt;
+            &lt;/div&gt;
+            &lt;div class="boxInput"&gt;
+                &lt;label for="password" class="boxLabel"&gt;Password&lt;/label&gt;
+                &lt;input type="password" id="password" class="itemInput fs-l"&gt;
+            &lt;/div&gt;
+        &lt;/div&gt;
```
Comment (kjais1720) [inline Templates/Inputs/inputs.html]: You can use JS to just replace all the `<` in the pre container with `&LT`, then you can write plain HTML

Comment (Sameony) [reply Templates/Inputs/inputs.html]: alright, will see to it

File: Templates/Inputs/inputs.html (hunk 2 above)
Comment (Sameony) [reply Templates/Inputs/inputs.html:8]: i was just going for the theme like:
mirinda cards
mirinda inputs
mirinda buttons and stuff-

File: Styles/main.css (hunk 1 above)
Comment (Sameony) [reply Styles/main.css]: updated :D

**Review by kjais1720 — APPROVED** [review]
All good then👍

**Pull request merged**